x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/ilm/ilm.ts

```diff
@@ -13,26 +13,3 @@ export function getIndexWithWithAlias(aliasName: string) {
     },
   };
 }
-
```

Comment:
@ruflin Not sure what this "getIndexWithWithAlias" function is doing.  Can it be removed?
https://github.com/elastic/kibana/pull/58869/commits/ee008082cb2f91d7ea5e9bc867ca8650d6d98516#diff-2f7a49f58cea808667dd6faca14721ebL7